src/test/regress/expected/distributed_functions.out

```diff
@@ -25,6 +25,13 @@ SELECT create_distributed_table('notices', 'id');
 (1 row)
 
 INSERT INTO notices VALUES (1, 'hello world');
+-- Create the necessary test utility function
+CREATE FUNCTION master_metadata_snapshot()
```

Comment:
minor: it is always more robust to have `CREATE OR REPLACE` just in case a test above this creates the same function